doc/index.rst

```diff
@@ -181,6 +181,31 @@ This represents the Tailwind configuration file:
     symfonycasts_tailwind:
         config_file: 'tailwind.config.js'
 
+If you include any other files containing classes to be detected outside the 
+default directories, like the `form themes provided with the Twig Bridge component <https://github.com/symfony/symfony/blob/HEAD/src/Symfony/Bridge/Twig/Resources/views/Form/tailwind_2_layout.html.twig/>`_, 
+untouched from the ``vendor`` directory, for instance, after changing your Twig configuration like this:
+
+.. code-block:: yaml
+
+    # config/packages/twig.yaml
+    twig:    
+        form_themes:
+            - 'tailwind_2_layout.html.twig'
+
+...you can add them to your ``tailwind.config.js`` file, like this:
```

Comment:
```suggestion
You can add them to your ``tailwind.config.js`` file like this:
```

Reply:
It's supposed to be a continuation from the unfinished sentence started above. Maybe that use of punctuation is not English. Is it?

Reply:
yeah, I see... I think with this simple way it will be better anyway